packages/common/src/core/types/IHubFeedback.ts

```diff
@@ -0,0 +1,13 @@
+import { IWithPermissions, IWithSlug } from "../traits";
+import { IHubItemEntity, IHubItemEntityEditor } from "./IHubItemEntity";
+
+/**
+ * Defines properties of Hub Feedback object
+ * @internal
+ */
+export interface IHubFeedback
+  extends IHubItemEntity,
+    IWithPermissions,
+    IWithSlug {}
```

Comment:
We don't _currently_ support slugs for Feedback items, and since we won't be supporting creating Feedback items in Hub any longer, we can probably drop `IWithSlug` here